src/main/java/com/kuebiko/it/bibhusan/NextPerfectSquare.java

```diff
@@ -0,0 +1,32 @@
+package com.kuebiko.it.bibhusan;
+
+/**
+ * Given a number, find if it's perfect square, if not perfect return -1
+ * if perfect find the next perfect square
+ *
+ * eg. 16 -> 25, 23 -> -1
+ *
+ */
+public class NextPerfectSquare {
+
+  public int getNextPerfectSquare(int num) {
+    if(!isPerfectSquare(num)){
+      return -1;
+    }
+    else{
+      while (true){
```

Comment:
Try this program with a do while, even though your program works while(true) is not a good coding practice